styles/rentry.co/catppuccin.user.css

```diff
@@ -0,0 +1,578 @@
+        color: @text !important;
+      }
+    }
+    .btn-success {
+      &,
+      &.btn,
+      &:hover {
+        color: @green !important;
+      }
+      &#submitButton {
+        background: @base !important;
+        &:hover {
+          background: @surface0 !important;
+        }
+      }
+    }
+    .btn-danger {
+      &,
+      &.btn,
+      &:hover {
+        color: @red !important;
+      }
+    }
+    .clipboard {
+      color: @text;
+      background: none;
+    }
+    #progressBar {
+      background-color: @blue !important;
+    }
+    .full-contrast:not(.active) path {
+      fill: @overlay2 !important;
+    }
+    .full-contrast.active path {
+      fill: @peach !important;
+    }
+    .edit-code {
+      background: @mantle;
+      color: @text !important;
+    }
+    .cb-tooltiptext {
+      color: @text !important;
+      background: @surface0 !important;
+    }
+    .cb-tooltiptext-bottom::after {
+      border-color: transparent transparent @surface0;
+    }
+    .headerlink {
+      color: @overlay1 !important;
+    }
+    #submitButton {
+      background: @mantle !important;
+    }
+    // claim-guide page
+    .color-change {
+      &[style="color:grey"] {
+        color: @overlay2 !important;
+      }
+      &[style="color:#039205"] {
+        color: @green !important;
+      }
+    }
+    // delete modal
+    .modal-content {
+      background: @crust;
+      color: @text;
+    }
+    .modal-body {
+      color: @overlay2;
+    }
+    .modal-header {
+      color: @text;
+    }
+    .modal-header,
+    .modal-footer {
+      border-color: @surface1 !important;
+    }
+    #deleteButton {
+      color: @red !important;
+    }
+    // footer
+    a {
+      color: @blue;
+
+      &:hover {
+        color: @sky !important;
+      }
+    }
+    .text-muted {
+      color: @overlay2 !important;
+    }
+    .text-primary {
+      color: @blue !important;
+    }
+    .text-success {
+      color: @green !important;
+    }
+    .text-warning {
+      color: @yellow !important;
+    }
+    .text-danger {
+      color: @red !important;
+    }
+    // editor
+    .CodeMirror-line::selection,
+    .CodeMirror-line > span::selection,
+    .CodeMirror-line > span > span::selection {
+      background-color: fade(@overlay2, 25%);
+    }
+    .cm-formatting-admonition {
+      color: @yellow !important;
+    }
+    .cm-mark {
+      color: @yellow !important;
+    }
+    .cm-link {
+      color: @blue !important;
+    }
+    .cm-url {
+      color: @sky !important;
+    }
+    .cm-formatting-toc {
+      color: @mauve !important;
+    }
+    .cm-variable-2 {
+      color: @green !important;
+    }
+    .cm-keyword {
+      color: @pink !important;
+    }
+    .cm-meta {
+      color: @subtext0 !important;
+    }
+    .cm-s-default .cm-comment {
+      color: @pink !important;
+    }
+    .cm-hr {
+      color: @surface2 !important;
+    }
+    // markdown
+    .cm-header {
+      color: @text !important;
+    }
```

Comment:
Instead of `!important`-ing everything, why not just wrap it in `.cm-s-default`? That should raise the specificity high enough, and you already do it for `cm-comment`.

Reply:
Also, `.cm-tag` is unthemed. Should be blue. You can get it by putting a `</div>` on its own line.